lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwift.h

```diff
@@ -198,13 +199,12 @@ class TypeSystemSwift : public TypeSystem {
   ConstString DeclContextGetScopeQualifiedName(void *opaque_decl_ctx) override {
     return {};
   }
-  bool
-  DeclContextIsClassMethod(void *opaque_decl_ctx,
-                           lldb::LanguageType *language_ptr,
-                           bool *is_instance_method_ptr,
-                           ConstString *language_object_name_ptr) override {
+  bool DeclContextIsClassMethod(void *opaque_decl_ctx) override {
     return false;
   }
+  lldb::LanguageType DeclContextGetLanguage(void *) override {
+    return lldb::eLanguageTypeUnknown;
```

Comment:
Why don't we need an implementation here? Because it's only used to differentiate between ObjC and C++?

Reply:
Should it return Swift, because all Swift DeclContexts are Swift?

Reply:
I will change it because it makes more sense that way. However, as it stands today, no valid Swift decl contexts are created, and so this method is never called. That's why I stubbed it out.